packages/performance-tests/test/TransformerRegression.test.ts

```diff
@@ -0,0 +1,127 @@
+const outputDir = path.join(__dirname, ".output");
+
+const setupTestData = async () => {
+  const logLevel = +process.env.LOG_LEVEL!;
+  if (LogLevel[logLevel] !== undefined) {
+    Logger.initializeToConsole();
+    Logger.setLevelDefault(LogLevel.Error);
+    Logger.setLevel(loggerCategory, LogLevel.Info);
+    Logger.setLevel(TransformerLoggerCategory.IModelExporter, logLevel);
+    Logger.setLevel(TransformerLoggerCategory.IModelImporter, logLevel);
+    Logger.setLevel(TransformerLoggerCategory.IModelTransformer, logLevel);
+  }
+  let usrEmail;
+  let usrPass;
+  if(process.env.V2_CHECKPOINT_USER_NAME){
+    usrEmail = process.env.V2_CHECKPOINT_USER_NAME;
+    usrPass = process.env.V2_CHECKPOINT_USER_PASSWORD;
+  } else if(process.env.V1_CHECKPOINT_USER_NAME){
+    usrEmail = process.env.V1_CHECKPOINT_USER_NAME;
+    usrPass = process.env.V1_CHECKPOINT_USER_PASSWORD;
+  } else {
+    usrEmail = process.env.IMODEL_USER_NAME;
+    usrPass = process.env.IMODEL_USER_PASSWORD;
+  }
+
+  assert(usrEmail, "user name was not configured");
+  assert(usrPass, "user password was not configured");
+  const user = {
+    email: usrEmail,
+    password: usrPass,
+  };
+
+  assert(process.env.OIDC_CLIENT_ID, "");
+  assert(process.env.OIDC_REDIRECT, "");
+  const authClient = process.env.CI === "1"
+    ? new TestBrowserAuthorizationClient({
+      clientId: process.env.OIDC_CLIENT_ID,
+      redirectUri: process.env.OIDC_REDIRECT,
+      scope: "itwins:read imodels:read imodels:modify",
+      authority: "https://qa-ims.bentley.com",
+    }, user)
+    : new NodeCliAuthorizationClient({
+      clientId: process.env.OIDC_CLIENT_ID,
+      redirectUri: process.env.OIDC_REDIRECT,
+      scope: "imodelaccess:read storage:modify realitydata:read imodels:read library:read imodels:modify realitydata:modify savedviews:read storage:read library:modify itwinjs savedviews:modify",
+    });
+
+  await authClient.signIn();
+
+  const hostConfig  = new IModelHostConfiguration();
+  hostConfig.authorizationClient = authClient;
+  const hubClient = new IModelsClient({ api: { baseUrl: `https://${process.env.IMJS_URL_PREFIX}api.bentley.com/imodels` } });
+  hostConfig.hubAccess = new BackendIModelsAccess(hubClient);
+  await IModelHost.startup(hostConfig);
+
+  const arrImodels = [];
+  for await (const iModel of getTestIModels()) {
+    arrImodels.push(iModel);
+  }
+  return arrImodels;
+};
+
+void (async function () {
+  const testIModels = await setupTestData();
+  var reporter = new Reporter();
+  const reportPath = initOutputFile("report.csv", outputDir);
+
+  // could probably add an outer describe here (surounding both foreaches) for any before after logic that isn't related to getting the test data. but idk for sure 
+  describe('Transformer Regression Tests', function () {
+    testIModels.forEach(async (value, index, _array) => {
+      const iModel = value;
+      if (index === 2 || index === 5) {
```

Comment:
can you use a more self-documenting identifier for the iModels instead of their index? Like matching on names? Also maybe we can move those identifiers into the .env of some other external test configuration since it won't make sense for external users trying to run these tests?